contracts/smart-account/modules/PasskeyValidationModules/Secp256r1.sol

```diff
@@ -358,7 +358,7 @@ library Secp256r1 {
             mstore(add(freemem, 0x80), _exp)
             mstore(add(freemem, 0xa0), _mod)
 
-            let success := staticcall(1500, 0x5, freemem, 0xc0, freemem, 0x20)
+            let success := staticcall(not(0), 0x5, freemem, 0xc0, freemem, 0x20)
```

Comment:
what does not(0) do?
shouldn't it be just 0

Reply:
The not is just the bitwise NOT operation.
`not(0) - not(000....0) -> (111....11) -> max int`

The second argument represents the gas to be used for the call. So, using not(0) essentially provides the maximum amount of gas possible.

Reply:
got it